connectivity/FEATURE_BLE/source/common/ble_trace_helpers.h

```diff
@@ -44,11 +44,38 @@ static inline const char* tr_as_array(T item)
     return (mbed_trace_array)((const uint8_t*)&item, sizeof(item));
 }
 
-static inline const char* to_string(UUID uuid)
-{
+static inline const char* to_string(const UUID &uuid) {
     return (mbed_trace_array)(uuid.getBaseUUID(), uuid.getLen());
 }
 
+static inline char* trace_uuid(const UUID& uuid)
```

Comment:
please don't static inline this

Reply:
I will consolidate the trace helpers into declarations (header) and definitions (source). This will resolve the `static inline` business.